plugins/powershell/plugin.spec.yaml

```diff
@@ -73,29 +79,46 @@ actions:
         description: PowerShell script as base64
         type: bytes
         required: true
+        example: R2V0LURhdGU=
       address:
         title: Address
         description: IP address of the remote host e.g. 192.168.1.1. If address is
           left blank PowerShell will run locally
         type: string
         required: false
+        example: 10.0.1.15
       host_name:
         title: Host Name
         description: Case-sensitive name of the remote host, eg. MyComputer for Kerberos
           connection only
         type: string
         required: false
+        example: windows
+      secret_key:
```

Comment:
We really dont want that, that exposes customers credentials in the UI

Reply:
@jkaluza-rapid7 This branch was not rebased properly as it was waiting a long time to be merged - changes from other commits where visible on the GH. Right now there are no credentials in the actions. All of them are moved to the connection.